storage/src/vespa/storage/common/statusmetricconsumer.cpp

```diff
@@ -62,85 +60,7 @@ StatusMetricConsumer::getReportContentType(const framework::HttpUrlPath& path) c
         return "application/json";
     }
 
-    return "text/html";
-}
-
-namespace {
-    void addSnapshotAsTableRow(const metrics::MetricSnapshot& snapshot,
-                               const metrics::MetricSnapshotSet* building,
-                               std::ostream& out,
-                               int32_t interval = -3)
-    {
-        if (interval == -3) interval = snapshot.getPeriod();
-        std::string name(snapshot.getName());
-        if (interval == -1) {
-            name = "Clone of total metrics with active metrics added";
-        }
-        std::vector<char> buffer(40);
-        out << "  <tr>\n"
-            << "    <td>" << name << "</td>\n";
-        //if (snapshot.getToTime() != 0 || interval < 0 || building != 0)
-        for (uint32_t format=0; format<4; ++format) {
-                // 0 XML - 1 HTML - 2 text, 3 JSON
-            out << "    <td>";
-            bool linked = false;
-            for (uint32_t tmp=0; tmp<2; ++tmp) { // 0 last - 1 temp
-                if (tmp == 1) {
-                    if (building == 0 || !building->hasTemporarySnapshot()
-                        || building->getBuilderCount() == 0)
-                    {
-                        continue;
-                    }
-                } else {
-                    if ((snapshot.getToTime() == 0 && interval >= 0)
-                        || snapshot.getToTime() == snapshot.getFromTime())
-                    {
-                        continue;
-                    }
-                }
-                if (tmp == 1) out << "&nbsp;&nbsp;";
-
-                const char* formatStr = "xml";
-                const char* consumer = "status";
-                switch (format) {
-                case 0:
-                    formatStr = "xml";
-                    break;
-                case 1:
-                    formatStr = "html";
-                    break;
-                case 2:
-                    formatStr = "text";
-                    break;
-                case 3:
-                    formatStr = "json";
-                    consumer = "yamas";
-                    break;
-                }
-
-                linked = true;
-                out << "<a href=\""
-                    << "?interval=" << interval
-                    << "&format=" << formatStr
-                    << "&consumer=" << consumer
-                    << "&verbosity=0"
-                    << "&pattern=.*"
-                    << "&callsnapshothooks=0"
-                    << "&tmpsnapshot=" << tmp
-                    << "\">"
```

Comment:
It would be good to have _some_ form of output on the main metrics page that gives the parameters to use for text metrics dump of the past 1 minute, 5 minutes and all-time. This can be hard-coded. It's just hard to remember the URLs by heart...!

Reply:
I will merge this one now, and we can discuss the details for a fixed text tomorrow.